src/main/java/sirius/biz/codelists/LookupValue.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.biz.codelists;
+
+import sirius.kernel.di.std.Part;
+
+/**
+ * Represents a string value backed by a {@link LookupTable} to be used in database entities.
+ * <p>
+ * Being a string value, {@link sirius.db.mixing.annotations.Length} and {@link sirius.db.mixing.annotations.NullAllowed}
+ * can be used as to further annotate a database field.
+ */
+public class LookupValue {
+
+    private final String lookupTableName;
+    private LookupTable table;
+
+    private final Display display;
+    private final Export export;
+    private final CustomValues customValues;
+    private String value;
+
+    /**
```

Comment:
Just a design question: should those in-class enums be defined at the end of the class?

Reply:
we generally keep them at the top

Reply:
Yes public ones at the top, private ones at the bottom in my opinion (as public ones are relevant for API and private ones are just implementation details like getters and setters are)